app/routes/events/components/EventAttendeeStatistics.css

```diff
@@ -27,5 +27,21 @@
 }
 
 .chartContainer {
+  margin-top: 2em;
   overflow: hidden;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 1em;
+}
+
+.chartCard, .graphCard {
+  background-color: var(--additive-background);
+}
```

Comment:
Sure, is this better?
| Dark mode | Light mode |
| -- | -- |
| ![Screenshot from 2023-09-29 13-05-32](https://github.com/webkom/lego-webapp/assets/33326578/7f479238-c0a7-4251-a3c3-08a2bdd930d7) | ![Screenshot from 2023-09-29 13-10-10](https://github.com/webkom/lego-webapp/assets/33326578/87080130-8141-4d4c-b8ad-e5a59a44db7f) |

Reply:
I wouldn't say they are better purely in terms of looks, but I'd say it's better since these will now "follow" changes in the card component. Nevertheless, this is an improvement of what they used to look like haha. Good job!

Also, I guess it's worth mentioning that we've sort of planning on removing the background of all pages, and I guess whenever that happens these would look much better.